zerver/lib/actions.py

```diff
@@ -376,7 +376,7 @@ def stream_name(stream):
                           default_events_register_stream=default_events_register_stream_name,
                           default_all_public_streams=user_profile.default_all_public_streams,
                           avatar_url=avatar_url(user_profile),
-                          owner=user_profile.bot_owner.email,
+                          owner=user_profile.bot_owner.email if user_profile.bot_owner else None,
                           ))
```

Comment:
Minor but I think it will be good to have it like this:
```
                            default_all_public_streams=user_profile.default_all_public_streams,
                          avatar_url=avatar_url(user_profile),
                          ))
    # Set the owner key only when the bot has an owner.
    # The deault bots don't have an owner. So don't
    # set the owner key while reactivating them.
    if user_profile.bot_owner is not None:
        event['bot']['owner'] = user_profile.bot_owner.email
    send_event(event, bot_owner_userids(user_profile))
```